katas/main/src/pokerhands/juan/PokerHands.cs

```diff
@@ -0,0 +1,437 @@
+﻿namespace PokerHands.Juan
+{
+    public class Card
+    {
+        private readonly char suit;
+        private readonly char value;
+        private readonly int weigth;
+
+        public Card(string input)
+        {
+            value = input[0];
+            suit = input[1];
+            weigth = ParsedValue(value);
+        }
+
+        public char SuitAttr => suit;
+        public char ValueAttr => value;
+        public int WeigthAttr => weigth;
+
+        private int ParsedValue(char value)
+        {
+            char[] values = { 'A', '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K' };
+            return Array.IndexOf(values, value) + 1;
+        }
+
+        public string ToStr()
+        {
+            return string.Format("suit:{0} value:{1} weigth:{2}", suit, value, weigth);
+        }
+    }
+
+    public static class GetHandType
```

Comment:
For class names, please use nouns and try to avoid verbs for the first word, this is the normal convention so it is easier to identify classes and methods.

In this case, a possible name could be `HandType` or something similar